socorro/collector/crashmover_app.py

```diff
@@ -103,5 +108,101 @@ def transform(self, crash_id):
                 exc_info=True
             )
 
+
+#==============================================================================
+class RawAndProcessedCopierApp(FetchTransformSaveApp):
+    """copy raw & processed crashes from a source to a destination"""
+    app_name = 'raw_and_processed_crash_copier'
+    app_version = '1.0'
+    app_description = __doc__
+
+    required_config = Namespace()
+    #--------------------------------------------------------------------------
+    # new_crash_source namespace
+    #     this namespace is for config parameter having to do with the source
+    #     of new crash_ids.
+    #--------------------------------------------------------------------------
+    required_config.namespace('new_crash_source')
+    required_config.new_crash_source.add_option(
+        'new_crash_source_class',
+        doc='an iterable that will stream crash_ids needing processing',
+        default='socorro.processor.timemachine.PGQueryNewCrashSource',
```

Comment:
We want to run this in parallel across multiple machines (and probably multiple threads per machine as well) - maybe we should make RabbitMQ the default (with a special queue name)